spectrum-network/src/protocol_handler/handel.rs

```diff
@@ -773,41 +775,42 @@ mod tests {
         own_peer: PeerId,
         peers: Vec<(PeerId, Option<Multiaddr>)>,
         contrib: Contrib,
+        conf: HandelConfig,
     ) -> Handel<Contrib, (), BinomialPeerPartitions<PseudoRandomGenPerm>> {
         let rng = PseudoRandomGenPerm::new([0u8; 32]);
         let pp = BinomialPeerPartitions::new(own_peer, peers, rng);
         let own_peer_ix = pp.try_index_peer(own_peer).unwrap();
-        Handel::new(CONF, contrib, (), pp, own_peer_ix)
+        Handel::new(conf, contrib, (), pp, own_peer_ix)
     }
 
-    #[test]
-    fn best_contrib_is_own_contrib_when_no_interactions() {
+    #[tokio::test]
+    async fn best_contrib_is_own_contrib_when_no_interactions() {
```

Comment:
Even though there's no async code in the test, the fact that `Handel` has a field from tokio::sleep means that this code will crash in non-async context.